qualification/recv.py

```diff
@@ -199,20 +200,60 @@ async def next_complete_chunk(
         ----------
         min_timestamp, max_delay
             See :meth:`complete_chunks`
+        timeout
+            Maximum time to wait
+
+        Raises
+        ------
+        TimeoutError
+            if a complete chunk is not received in time
+        RuntimeError
+            if the stream is stopped before a complete chunk is received
         """
         async with asyncio.timeout(timeout):
             async for timestamp, chunk in self.complete_chunks(min_timestamp=min_timestamp, max_delay=max_delay):
                 with chunk:
                     return timestamp, np.array(chunk.data)  # Makes a copy before we return the chunk
         raise RuntimeError("stream was shut down before we received a complete chunk")
 
+    async def wait_complete_chunk(
+        self,
+        min_timestamp: int | None = None,
+        *,
+        max_delay: int = DEFAULT_MAX_DELAY,
+        timeout: float | None = DEFAULT_TIMEOUT,
+    ) -> int:
+        """Wait for a complete chunk, but do not return it.
+
+        Only the timestamp is returned.
```

Comment:
Could I request a small update/bit of bikeshedding, considering both this and `next_complete_chunk` exist
* Perhaps a tweak/addition to this docstring as per your commit/PR blurb clarifying the difference between the two functions,
* Or even just why you/one would use one or the other

> [wait_complete_chunk] is more efficient that next_complete_chunk because it doesn't make and return a copy of the data.